.github/actions/bazel-test-all/action.yaml

```diff
@@ -121,3 +124,18 @@ runs:
             echo "Bazel version: $(bazel version)"
 
             bazel ${{ inputs.BAZEL_COMMAND }} "${bazel_args[@]}"
+
+      - name: Upload to S3
+        uses: ./.github/actions/bazel
+        if: inputs.upload-artifacts == 'true'
+        with:
+          run: |
+            release_build='${{ inputs.release-build }}'
+
+            if [[ $release_build == "true" ]]; then
+              echo "refusing to upload non-release artifacts"
+              exit 1
+            fi
+
+            echo uploading artifacts to CDN
+            bazel run --check_up_to_date //:upload-artifacts
```

Comment:
So [--check_up_to_date](https://bazel.build/docs/user-manual#check-up-to-date) expects `//:upload-artifacts` to have already been built. Is that guaranteed by the "Run Bazel Commands" above? If so, could you write that as a comment so future readers don't have to think about it?